cli50/__main__.py

```diff
@@ -57,7 +57,7 @@ def main():
     # Check for newer version
     try:
         latest = max(requests.get("https://pypi.org/pypi/cli50/json").json()["releases"])
-        assert latest <= __version__
+        assert pkg_resources.parse_version(latest) <= pkg_resources.parse_version(__version__)
```

Comment:
Afraid the value of `latest` would still be incorrect, since `max` doesn't take semantic versioning into account, per the `1.9.0` example!

Reply:
Oops! Sorry about that no sure myself why I did that!! 😅 [This](https://github.com/cs50/cli50/pull/79/commits/57a3ca2d6a2fb5cb392c8b280ddf032d8076706e) should resolve!